src/RichText/Toolbar/toolbarTheme.ts

```diff
@@ -80,13 +82,38 @@ export const defaultToolbarTheme: ToolbarTheme = {
       paddingHorizontal: 0,
     },
   },
+  section: {
+    flexDirection: 'row',
+  },
+  stickyKeyboardContainer: {
+    width: 44,
+    height: '100%',
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  flatList: {
+    flex: 1,
+  },
```

Comment:
The styling here is abit off 

![Screenshot 2024-09-10 at 16 21 56](https://github.com/user-attachments/assets/4984c97d-e044-4c13-95ee-e89eca275474)

![Screenshot 2024-09-10 at 16 23 21](https://github.com/user-attachments/assets/89bb4e7e-170e-4902-8d88-a261af2f6363)